README.md

```diff
@@ -148,6 +148,7 @@ The format of this glossary is modeled after [HugoGiraudel/SJSJ](https://github.
 * **Electrically-Erasable Programmable ROM**:
     _see [EEPROM](#eeprom)_
 * Electroluminescent wire:
+* Element: Name for common electrical component such as a resistor, capacitor, inductor, etc.
 * EL wire:
```

Comment:
I'll take care of it post-merge because it's trivial, but I wanted to let you know I'll be bolding _Element_, e.g.:

```
* **Element*: ...
```